lua/nvim-tree/actions/fs/trash.lua

```diff
@@ -30,16 +30,11 @@ function M.fn(node)
   end
 
   -- configs
-  if utils.is_unix then
```

Comment:
Can we please be conservative and use is_unix or is_windows ?

We'll sort out macos later: #2336

Reply:
update: use `is_unix or is_windows` check

Reply:
please check again, should be OK now.